app/src/mixins/update-user-mixin.ts

```diff
@@ -16,7 +16,7 @@ export class UpdateUserMixin extends Vue {
       }
     } catch (err) {
       // just log the error -- no need to halt app
-      console.log('Launch Darkly update error =', err) // eslint-disable-line no-console
+      console.log('Error updating user =', err) // eslint-disable-line no-console
```

Comment:
The error could be in `fetchUserInfo` or `updateLaunchDarkly` so I generalized this message.